x-pack/plugins/security_solution/public/flyout/document_details/right/components/highlighted_fields_cell.tsx

```diff
@@ -75,25 +79,30 @@ export interface HighlightedFieldsCellProps {
 /**
  * Renders a component in the highlighted fields table cell based on the field name
  */
-export const HighlightedFieldsCell: VFC<HighlightedFieldsCellProps> = ({ values, field }) => (
+export const HighlightedFieldsCell: VFC<HighlightedFieldsCellProps> = ({
+  values,
+  field,
+  originalField,
+}) => (
   <>
     {values != null &&
       values.map((value, i) => {
         return (
           <EuiFlexItem
             grow={false}
             key={`${i}-${value}`}
-            data-test-subj={`${value}-${HIGHLIGHTED_FIELDS_CELL_TEST_ID}`}
+            data-test-subj={`${field}-${HIGHLIGHTED_FIELDS_CELL_TEST_ID}`}
```

Comment:
curious about the reason for this change? I can see `field` being safer than `value` as we could have identical values but less likely identical fields. If that was the primary reason, why not change the `key` property the line above? `key` should be unique as well right?